package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "msbuild",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "msbuild wrapper",
   "main": "msbuild.js",
   "scripts": {
```

Comment:
Again, I am not sure this is right. I was under the impression that whenever we make small code fixes like this, we need to increase the patch version number. (See https://www.sitepoint.com/semantic-versioning-why-you-should-using/). As far as I see, the current nodejs-msbuild version is 1.0.1 (https://www.npmjs.com/package/msbuild). Thus, this patch should be 1.0.2.